rllib/evaluation/sampler.py

```diff
@@ -263,12 +263,21 @@ def _env_runner(base_env, extra_batch_callback, policies, policy_mapping_fn,
     """
 
     try:
-        if not horizon:
+        # If Trainer's horizon is provided, force-set the env's
+        # `max_episode_steps` spec to that value (otherwise, Trainer's horizon
+        # setting will be violated).
+        print("base_env={}".format(base_env))
```

Comment:
Use logger.debug() or info() here?

Reply:
Actually not sure if this should be logged here at all. I think it might already be logged at INFO elsewhere.

Reply:
Ah, sorry, leftover from debugging. I'll remove.